docs/fast_data/configuration/single_view_trigger_generator.md

```diff
@@ -88,3 +87,22 @@ The Kafka Projection Updates config map contains the `kafkaProjectionUpdates.jso
 
 Remember to copy/paste the mount path into the `KAFKA_PROJECTION_UPDATES_FOLDER` environment variable so the service can read the file.
 If you need more info on how to configure the `kafkaProjectionUpdates.json` file, please refer to the [Kafka Projection Updates](/fast_data/configuration/config_maps/kafka_projection_updates.md) page.
+
+## Error Codes
```

Comment:
May we add a reference to this error codes section also in the troubleshooting page?

Reply:
Shall I move the error codes directly within a custom page, where all of them can be found (e.g. Fast Data Error Codes)?

Reply:
I created a new page

Reply:
great! Let's add in the FAQ page the link to this error codes section. We can create a paragraph that says "if you want to better understand error codes go to this new page" or something like that. 

wdyt?

Reply:
Done. I also reviewed the error codes to match our troubleshooting guidelines